test/resources/serial_console.resource

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Library     PexpectLibrary
+Library    serial_console.py
+
+Resource    serial_console.resource
+
+
+*** Variables ***
+${PROMPT_PATTERN}   \\[["\\w"@\\-~\\ ]+\\]\\$\\ *
```

Comment:
What if we set the PS1 value for the redhat user in our images to simplify this?